### PR TITLE
EVG-12448 Remove oppressive terminology from the Evergreen codebase

### PR DESCRIPTION
**Pull request by bsamek**
https://tools.ietf.org/id/draft-knodel-terminology-00.html

### CODE REVIEW

File: model/task/task.go [hunk 1]
```diff
@@ -1313,16 +1313,16 @@ func (t *Task) UpdateHeartbeat() error {
 	)
 }
 
-// Blacklist sets the priority of a task so it will never run.
+// Block sets the priority of a task so it will never run.
 // It also deactivates the task and any tasks that depend on it.
 // The build cache is not updated
-func (t *Task) Blacklist(user string) ([]Task, error) {
-	t.Priority = evergreen.BlacklistPriority
+func (t *Task) Block(user string) ([]Task, error) {
```
Comment (ybrill) [inline model/task/task.go:1319]: this seems to be overloading the Blocked terminology
(until now it's just been for tasks with unsatisfied dependencies)

Comment (john-m-liu) [reply model/task/task.go:1319]: Agreed, can we call them disabled tasks?

Comment (bsamek) [reply model/task/task.go]: fixed

File: model/commitqueue/commit_queue.go [hunk 2]
```diff
@@ -275,7 +275,7 @@ func preventMergeForItem(patchType string, versionExists bool, item CommitQueueI
 			return errors.New("merge task doesn't exist")
 		}
 		event.LogMergeTaskUnscheduled(mergeTask.Id, mergeTask.Execution, user)
-		if _, err = mergeTask.Block(user); err != nil {
+		if _, err = mergeTask.SetBlockedPriority(user); err != nil {
```
Comment (ybrill) [inline model/commitqueue/commit_queue.go:278]: I still think it's overloading the blocked concept.

Comment (bsamek) [reply model/commitqueue/commit_queue.go]: Used "disabled" instead.

File: model/commitqueue/commit_queue.go [hunk 3]
```diff
@@ -266,7 +266,7 @@ func preventMergeForItem(patchType string, versionExists bool, item CommitQueueI
 			return errors.Wrap(err, "can't clear subscriptions")
 		}
 
-		// Blacklist the merge task
+		// Block the merge task
```
Comment (ybrill) [inline model/commitqueue/commit_queue.go:269]: should this be "Disable" too?

**Pull request merged**